CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+## 0.3.3 (January 14th, 2020)
+
```

Comment:
Any Security section is listed first in the Vault changelog, and I think we should copy that pattern here.

Reply:
Done!